docs/semgrep-supply-chain/overview.md

```diff
@@ -21,121 +21,112 @@ Object.entries(frontMatter).filter(
-
-Refer to [Supported languages](/docs/supported-languages#semgrep-supply-chain) to see all languages supported by Semgrep Supply Chain.
-
-## Transitive dependencies and reachability analysis
-
-See [SSC glossary > Transitivity](/docs/semgrep-supply-chain/glossary/#transitive-or-indirect-dependency) for a definition of a transitive dependency.
-
-* Semgrep Supply Chain does **not** perform reachability analysis for transitive dependencies. This means we do not scan the source code of your dependencies to determine if their dependencies may produce a reachable finding in the code.
-* Semgrep Supply Chain supports scanning for transitive or indirect dependencies for all of its [supported languages](/docs/supported-languages#semgrep-supply-chain). Findings are collected and displayed in **Semgrep Cloud Platform** > **Supply Chain**.
-* In most cases, Semgrep Supply Chain generates **reachable findings** for **direct dependencies**. However, there are certain dependencies that are vulnerable simply through their inclusion in a codebase. Semgrep Supply Chain generates reachable findings for these types of dependencies even if they are transitive dependencies.
-
-:::info
-In some package ecosystems, it's possible to use a transitive dependency in your code as if it were a direct dependency. This is very uncommon, but Semgrep does scan for these usages with reachability rules if they are available, and will mark transitive dependencies as "unreachable" if they are not directly used.
-:::
-
-## Next steps: Scanning your codebase
-
-To scan your codebase, follow the instructions in [Scanning open source dependencies](/semgrep-supply-chain/getting-started).
-
-## Additional references
-
-* [Software supply chain security is hard](https://r2c.dev/blog/2022/software-supply-chain-security-is-hard/)
-* [The best free, open-source supply-chain security tool? The lockfile](https://r2c.dev/blog/2022/the-best-free-open-source-supply-chain-tool-the-lockfile/)
+However, some dependencies are vulnerable simply through their inclusion in a
+codebase; in such cases, Semgrep Supply Chain generates reachable findings
+involving these dependencies, even if they're transitive, not direct,
+dependencies.
+
+Some package ecosystems allow the use of a transitive dependency as if it were a
+direct dependency. Though this feature is uncommon, Semgrep Supply Chain can
+scan for such usage and flag transitive dependencies as unreachable if not used
+directly.
+
+## Software bill of materials
+
+Semgrep Supply Chain can [generate a software bill of materials
+(SBOM)](/semgrep-supply-chain/sbom), a complete inventory of your
+third-party/open source components to assist you with your auditing procedures.
+
+## Dependency search
+
+Semgrep Supply Chain's dependency search feature allows you to query for
+dependencies in your codebase; it can detect direct and transitive dependencies
+in any repository you've scanned with. The results list the dependency, along
+with 
```

Comment:
> with ... ??? 

I think there is more here?

Reply:
I think it's doing that thing again where I don't have your latest work - I checked the latest commit, and there's still nothing here?
![falling-off-text](https://github.com/semgrep/semgrep-docs/assets/20766840/110ccf45-fa6a-4c35-9f57-844d08e7102f)